api/app/Aggregates/Events/EventAggregate.php

```diff
@@ -11,15 +11,15 @@ class EventAggregate
 {
     private int $id;
     private string $inviteeEmail;
-    private Carbon $startDate;
```

Comment:
I am not sure the file should be changed here. @yevhen-turovets should be still working on this.

Reply:
I change this line at last pull request